app/assets/stylesheets/homepage.css.scss

```diff
@@ -16,14 +16,39 @@
     font-weight: 200;
 }
 
-.nav-bar-nk {
+#logo {
+    text-decoration: none;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered margin-left, text-decoration